scripts/commit_classifier.py

```diff
@@ -69,18 +69,30 @@ def apply_phab(self, hg, diff_id):
         diffs = phabricator_api.search_diffs(
             diff_phid=[p[0] for p in patches], attachments={"commits": True}
         )
-        commits = {
-            diff["phid"]: diff["attachments"]["commits"].get("commits", [])
-            for diff in diffs
-        }
+
+        diffs_data = {}
+        for diff in diffs:
+            revision = phabricator_api.load_revision(rev_phid=diff["revisionPHID"])
+            logger.info(
+                "Diff {} linked to Revision {}".format(diff["id"], revision["id"])
+            )
+
+            diffs_data[diff["phid"]] = {
+                "commits": diff["attachments"]["commits"].get("commits", []),
+                "revision": revision,
+            }
 
         # First apply patches on local repo
         for diff_phid, patch in patches:
-            commit = commits.get(diff_phid)
+            diff_data = diffs_data.get(diff_phid)
+
+            commits = diff_data["commits"]
+            revision = diff_data["revision"]
 
-            message = ""
-            if commit:
-                message += "{}\n".format(commit[0]["message"])
+            if commits and commits[0]["message"]:
+                message = commits[0]["message"]
```

Comment:
You lost the `\n` here. Is that wanted ?

Reply:
Yeah, I thought it's actually unnecessary